fix: LEAP-388: Simplify SVG data URL generation

robot-ci-heartex opened the pull request:
`encodeSVG()` is too complex for expected outcome, so it was simplified.
- No quotes management
- No namespace control (we build that SVG, so have full control already)
- No other unnecessary code we don't need, because SVG is generated by us
- Comments
- Simple unit tests for couple of cases

### PR fulfills these requirements
- [x] Tests for the changes have been added/updated
- [ ] Docs have been added/updated
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance

### Describe the reason for change
https://github.com/HumanSignal/label-studio-frontend/security/code-scanning/10
Initial idea was to use SVG parser to parse+generate SVG again, but it turned out as a simplier problem

### What feature flags were used to cover this change?
None

### This change affects (describe how if yes)
- [ ] Performance
- [x] Security
- [ ] UX

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
- [ ] e2e (codecept)
- [ ] integration (cypress)
- [x] unit (jest)

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8331687392

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8331713791

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8389191023)
> Successfully merged:  90 files changed, 3079 insertions(+), 1080 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-388/svg-xmlns'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8389191023

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8389211613

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8453311990)
> Successfully merged:  75 files changed, 12234 insertions(+), 8580 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-388/svg-xmlns'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8453311990
